src/physics/utilities/equation_solver.hpp

```diff
@@ -17,8 +17,8 @@
 #include <optional>
 #include <variant>
 
-#include "common/serac_types.hpp"
 #include "mfem.hpp"
+#include "physics/utilities/solver_config.hpp"
 
 namespace serac {
 
```

Comment:
the equation solver is completely independent of the physics, so it feels wrong to describe it as a physics utility

Reply:
that being said, I don't really know where it should go either

Reply:
I view the `physics/utilities` as things that helps out with the discretization. However, I agree it isn't really a perfect fit. I think it's fine for now as a first pass.